### PR TITLE
adds paramedic labcoat

### PR DESCRIPTION
**Pull request by coldironwarrior**
## About The Pull Request

title

## Why It's Good For The Game

we have the icon file and i think it's one of the better-looking jackets and would like to enable it in the synth vendor

## Changelog
:cl:
add: adds the paramedic labcoat
/:cl:

### CODE REVIEW

**Comment by ivanmixo** [comment]
if its not obtainable it shouldnt be in the code

**Comment by ivanmixo** [comment]
then the changelog is bad...

File: code/modules/clothing/suits/labcoat.dm [hunk 1]
```diff
@@ -60,19 +60,18 @@
 	update_clothing_icon()
 
 /obj/item/clothing/suit/storage/labcoat/cmo
-	name = "chief medical officer's labcoat"
+	name = "Chief Medical Officer's labcoat"
```
Comment (TiviPlus) [inline code/modules/clothing/suits/labcoat.dm:63]: why are you upper casing this? byond automatically adds the as needed (ie eats the labcoat vs eats Labcoat)

File: code/modules/clothing/suits/labcoat.dm [hunk 2]
```diff
@@ -84,6 +83,11 @@
 	desc = "A suit that protects against minor chemical spills. Offers slightly more protection against biohazards than the standard model. Has a green stripe on the shoulder."
 	icon_state = "labcoat_viro"
 
+/obj/item/clothing/suit/storage/labcoat/genetics
+	name = "Geneticist labcoat"
```
Comment (TiviPlus) [inline code/modules/clothing/suits/labcoat.dm:87]: ditto

File: code/modules/clothing/suits/labcoat.dm [hunk 3]
```diff
@@ -60,19 +60,18 @@
...
 	name = "The Mad's labcoat"
 	desc = "It makes you look capable of konking someone on the noggin and shooting them into space."
 	icon_state = "labgreen"
 
-/obj/item/clothing/suit/storage/labcoat/genetics
-	name = "Geneticist labcoat"
-	desc = "A suit that protects against minor chemical spills. Has a blue stripe on the shoulder."
-	icon_state = "labcoat_gen"
+/obj/item/clothing/suit/storage/labcoat/paramedic
+	name = "Paramedic's labcoat"
```
Comment (TiviPlus) [inline code/modules/clothing/suits/labcoat.dm:72]: ditto

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/labcoat.dm (hunk 1 above)
Comment (coldironwarrior) [reply code/modules/clothing/suits/labcoat.dm:63]: i have a disease, it's called an english degree

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**